crates/torii/src/cli.rs

```diff
@@ -50,6 +122,21 @@ struct Args {
     start_block: u64,
 }
 
+fn address(arg: &Args, env_metadata: Option<&Environment>) -> Result<FieldElement, Error> {
```

Comment:
Seems like the same logic that `sozo` use to fetch `world_address` can also be applied here for `torii`. Maybe to keep things DRY we can import [`WorldOptions`](https://github.com/dojoengine/dojo/blob/2caa5537a248619526cd2f8e0c57025736aade74/crates/sozo/src/commands/options/world.rs#L11) for this.